### PR TITLE
new outtake class

### PR DESCRIPTION
**Pull request by anvigu**

### CODE REVIEW

File: src/main/java/frc/robot/commands/Outtake.java [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+import edu.wpi.first.wpilibj2.command.SubsystemBase;
+
+public class Outtake extends SubsystemBase {
+    /**
+     * Creates a new ExampleSubsystem.
+     */
+    public Outtake() {
+
+    }
+    public static void spin(){
```
Comment (MichaelCrandall) [inline src/main/java/frc/robot/commands/Outtake.java]: change spin to shoot

File: src/main/java/frc/robot/commands/Outtake.java [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+
+    }
+    public static void stop(){
```
Comment (MichaelCrandall) [inline src/main/java/frc/robot/commands/Outtake.java]: What do you mean exactly by stop? like not shoot?

Comment (anvigu) [reply src/main/java/frc/robot/commands/Outtake.java]: yes, i mean like stop spinning

Comment (anvigu) [reply src/main/java/frc/robot/commands/Outtake.java]: i changed it to stopShoot

File: .idea/workspace.xml [hunk 3]
```diff
@@ -34,15 +34,15 @@
     <property name="SHARE_PROJECT_CONFIGURATION_FILES" value="true" />
     <property name="last_opened_file_path" value="$PROJECT_DIR$" />
   </component>
-  <component name="ServiceViewManager">
-    <option name="viewStates">
```
Comment (SOR-FRCTeam6059) [inline .idea/workspace.xml]: dont commit changes to the .xml file

**Review by SOR-FRCTeam6059 — DISMISSED** [review]

**Pull request merged**